File: src/main/java/com/example/temperatura/converter/interfaces/CalculadoraTemperatura.java
package com.example.temperatura.converter.interfaces;

public interface CalculadoraTemperatura {

	Double celsiusToFarenheit(Double tempCelsius);

	Double celsiusToKelvin(Double tempCelsius);

	Double farenheitToCelsius(Double tempFarenheit);

	Double farenheitToKelvin(Double tempFarenkeit);

	Double kelvinToCelsius(Double tempKelvin);

	Double kelvinToFarenheit(Double tempKelvin);

}

File: src/main/kotlin/com/example/temperatura/converter/interfaces/CalculadoraTemperatura.kt
package com.example.temperatura.converter.interfaces

interface CalculadoraTemperatura {
    fun celsiusToFarenheit(tempCelsius: Double): Double
    fun celsiusToKelvin(tempCelsius: Double): Double
    fun farenheitToCelsius(tempFarenheit: Double): Double
    fun farenheitToKelvin(tempFarenheit: Double): Double
    fun kelvinToCelsius(tempKelvin: Double): Double
    fun kelvinToFarenheit(tempKelvin: Double): Double
}